config.py

```diff
@@ -47,6 +47,8 @@
 DIRECT_SUBMISSION = os.environ.get('DIRECT_SUBMISSION', False)
 BIOSAMPLES_URL = os.environ.get('BIOSAMPLES_URL', 'https://wwwdev.ebi.ac.uk/biosamples')
 
+BIOSTUDIES_ENV = 'dev'
```

Comment:
Can this be overridden by an env variable as well?

Reply:
@amnonkhen if you would read the PR description, that you would have your answer 😉

Reply:
It is just a temp variable and I hope we can remove it maybe today in another PR